backend/models/location.js

```diff
@@ -0,0 +1,7 @@
+const { Schema, model } = require('mongoose')
+
+// User Schema
+const LocationSchema = new Schema({
```

Comment:
we could store `{ latitude, longitude }` from the html5 geo api, on the user
and when we retrieved country, city, zip also store that